model/metrics/gen-ai.yaml

```
@@ -0,0 +1,39 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'This group describes GenAI metrics attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+      - ref: server.address
+        requirement_level: required
```

Given that you can run genai locally and prior history of not being able to obtain server address:port for databases and messaging systems, I think we can't require it

```suggestion
        requirement_level: recommended
```